### ISSUE TITLE
Treebuilder fails with cryptic error

### ISSUE THREAD

**Issue by pgcudahy** [tseemann/nullarbor#179]
Hello, just installed nullarbor for the first time and tried running it on 44 samples I have. The run command was:
`nice make -j 5 -l 16 -C results 2>&1 | tee -a results/nullarbor.log`
It went well for several hours until this:
```
aln="core.aln" tree="core.newick" cpus=4 opts="" /home/pgcudahy/ngs-tools/nullarbor/bin/../plugins/treebuilder/iqtree_fast.sh
snp-dists -b core.aln > distances.tab
This is snp-dists 0.6
Read 45 sequences of length 3922
/home/pgcudahy/ngs-tools/nullarbor/bin/../plugins/treebuilder/iqtree_fast.sh: 4: set: Illegal option -o pipefail
Makefile:130: recipe for target 'core.newick' failed
make: *** [core.newick] Error 2
make: *** Waiting for unfinished jobs....
```
I tried the make command again, but again got:
```
aln="core.aln" tree="core.newick" cpus=4 opts="" /home/pgcudahy/ngs-tools/nullarbor/bin/../plugins/treebuilder/iqtree_fast.sh
roary2svg.pl roary/gene_presence_absence.csv > roary/pan.svg
/home/pgcudahy/ngs-tools/nullarbor/bin/../plugins/treebuilder/iqtree_fast.sh: 4: set: Illegal option -o pipefail
Makefile:130: recipe for target 'core.newick' failed
make: *** [core.newick] Error 2
make: *** Waiting for unfinished jobs....
```

I've tried googling around but can't find anything relevant to that error. Any ideas of where this went wrong?

**Comment by pgcudahy** [comment]
I think the error might be that the iqtree_fast.sh starts with #!/bin/sh, which on my ubuntu server is a simlink to dash (the standard debian shell). iqtree_fast.sh has a reference to common.inc. Looking in common.inc, on line 4 I see
`set -eu -o pipeline`
and "-o pipeline" doesn't work in dash. A fix might be to explicitly invoke bash with #!/bin/bash at the top of iqtree_fast.sh, since "-o pipeline" is bash specific. (source at https://sipb.mit.edu/doc/safe-shell/)

After removing -o pipeline I got another error of 
`Ultrafast bootstrap does not work with -fast, -te or -n option`
which I think is unrelated and I fixed by again editing iqtree_fast.sh to remove the -fast option. Now everything works.

I'm not sure if these files are specific to nullarbor or part of iqtree or some other component. If nullarbor is the right place I could send a pull request with the modified iqtree_fast.sh
